server/views/pages/booking.njk

```diff
@@ -0,0 +1,54 @@
+{% extends "../partials/layout.njk" %}
+
+{% set pageTitle = "Bookings" %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">Bookings</h1>
+
+      <p>Only bookings made using GOV.UK One Login are shown.</p>
+
+
+      {% if visits | length %}
+        <form action="{{ paths.BOOKINGS }}" method="POST" novalidate>
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+
+        <h2 class="govuk-heading-m">Future visits</h2>
+
+        {% for visit in visits %}
+          <div class="bookings-content-block">
+            <h3 class="govuk-heading-s" data-test="visit-date">{{ visit.startTimestamp | formatDate(dateFormats.PRETTY_DATE) }}</h3>
```

Comment:
```suggestion
            <h3 class="govuk-heading-s" data-test="visit-date-{{ loop.index }}">{{ visit.startTimestamp | formatDate(dateFormats.PRETTY_DATE) }}</h3>
```

Might be worth suffixing all the `data-test` attributes within this loop with `loop.index` to make testing easier? This way each visit in the list would have a unique set of test attributes.